scalability_test/test_resources.yaml

```diff
@@ -0,0 +1,192 @@
+    labels:
+      ope-test-run: ${TEST_RUN_NAME}
+      app: ${NOTEBOOK_NAME}
+      opendatahub.io/dashboard: 'true'
+      opendatahub.io/odh-managed: 'true'
+      opendatahub.io/user: ${USERNAME}
+  spec:
+    template:
+      spec:
+        affinity:
+          nodeAffinity:
+            preferredDuringSchedulingIgnoredDuringExecution:
+              - preference:
+                  matchExpressions:
+                    - key: nvidia.com/gpu.present
+                      operator: NotIn
+                      values:
+                        - 'true'
+                weight: 1
+        containers:
+          - resources:
+              limits:
+                cpu: '1'
+                memory: 2Gi
+              requests:
+                cpu: '1'
+                memory: 2Gi
+            readinessProbe:
+              failureThreshold: 3
+              httpGet:
+                path: /notebook/ope-rhods-testing-1fef2f/${NOTEBOOK_NAME}/api
+                port: notebook-port
+                scheme: HTTP
+              initialDelaySeconds: 10
+              periodSeconds: 5
+              successThreshold: 1
+              timeoutSeconds: 1
+            name: ${NOTEBOOK_NAME}
+            livenessProbe:
+              failureThreshold: 3
+              httpGet:
+                path: /notebook/ope-rhods-testing-1fef2f/${NOTEBOOK_NAME}/api
+                port: notebook-port
+                scheme: HTTP
+              initialDelaySeconds: 10
+              periodSeconds: 5
+              successThreshold: 1
+              timeoutSeconds: 1
+            env:
+              - name: NOTEBOOK_ARGS
+                value: |-
+                  --ServerApp.port=8888
+                                    --ServerApp.token=''
+                                    --ServerApp.password=''
+                                    --ServerApp.base_url=/notebook/ope-rhods-testing-1fef2f/${NOTEBOOK_NAME}
+                                    --ServerApp.quit_button=False
+                                    --ServerApp.tornado_settings={"user":"${USERNAME}","hub_host":"https://rhods-dashboard-redhat-ods-applications.apps.shift.nerc.mghpcc.org","hub_prefix":"/projects/ope-rhods-testing-1fef2f"}
+              - name: JUPYTER_IMAGE
+                value: >-
+                  image-registry.openshift-image-registry.svc:5000/redhat-ods-applications/base-ope-image:latest
+            ports:
+              - containerPort: 8888
+                name: notebook-port
+                protocol: TCP
+            imagePullPolicy: Always
+            volumeMounts:
+              - mountPath: /opt/app-root/src
+                name: ${NOTEBOOK_NAME}
+              - mountPath: /dev/shm
+                name: shm
+            image: >-
+              image-registry.openshift-image-registry.svc:5000/redhat-ods-applications/base-ope-image:latest
+            workingDir: /opt/app-root/src
+          - resources:
+              limits:
+                cpu: 100m
```

Comment:
Is there any specific reason for choosing such a small size for oauth-proxy container?

Reply:
Probably to avoid unnecessarily impacting a project's resource quota (but that's just a guess).